twilio-connector/twilio_connector.py

```diff
@@ -0,0 +1,145 @@
+from curses.ascii import isdigit
+import os
+import argparse
+import pandas as pd
+from ratelimit import limits, sleep_and_retry
+from collections import defaultdict
+from cnvrgv2 import Cnvrg
+from twilio.rest import Client
+
+# The rate limit based on Twilio's rate of dequeueing messages
+# For details, refer to https://support.twilio.com/hc/en-us/articles/115002943027-Understanding-Twilio-Rate-Limits-and-Message-Queues#h_01F0265SAZ9BYNPG0AQZ0AKRHZ
+NUM_OF_CALLS = 3
```

Comment:
Is it possible to pass these global var via config files? 

Okay now I see that these are default func arguments.